.github/workflows/test-job.yaml

```diff
@@ -144,7 +144,7 @@ jobs:
         working-directory: ${{env.working-directory}}
 
       - name: Test
-        run: go test ./...
+        run: go test ./pkg/... ./cmd/... -race -parallel 4
```

Comment:
Could `make test` with proper `GO_TEST_FLAGS` set be called so the packages is defined in a single location.